### ISSUE TITLE
Updated analysis: EWS fusion summary and subtyping: remove EWSR1--PATZ1 and update To be classified status

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#825]
#### What analysis module should be updated and why? What changes need to be made? Please provide enough detail for another participant to make the update.
- [Fusion summary](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/fusion-summary/results/fusion_summary_ewings_foi.tsv) to remove `EWSR1--PATZ1` 
  - [This paper ](https://pubmed.ncbi.nlm.nih.gov/29679497/)suggests that glialneuronal brain tumors can harbor this fusion, so it cannot be considered EWS-specific. This affects only one tumor, which is noted as a glioblastoma in `pathology_free_text_diagnosis` in v17, confirming it should be removed from this module/classification.
- Remove `EWS, To be classified` status and replace with `EWS` based on pathology_diagnosis. 
  - The samples annotated as `To be classified` were done so because the canonical fusions were not detected. However, we should specify the rules such that 
    1) If EWS in pathology_diagnosis, annotate as EWS 
    2) If another pathology_diagnosis, but harbors unique EWS fusion, annotate as EWS

#### What input data should be used? Which data were used in the version being updated?
```
v17 pbta-fusion-putative-oncogenic.tsv
v17 fusion_summary_ewings_foi.tsv
```

#### When do you expect the revised analysis will be completed?
within a week

#### Who will complete the updated analysis?
@jharenza 

Note @jaclyn-taroni - this will update two modules, but since the updates are very minor, I put this together in one ticket and PR.

**Comment by jharenza** [comment]
closed with #826
